source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,160 @@
+Native Instruments Traktor Kontrol S3
+=====================================
+
+.. sectionauthor::
+  Owen Williams <owilliams at mixxx.org>
+
+.. versionadded:: 2.3.0
+
```

Comment:
The other mappings have this at the bottom of the first section, i.e. in line 19. This makes sense if we later add `versionchanged`, so please move it.

Reply:
done

Reply:
Nope, not resolved yet.

Reply:
sorry, bad push